### DISCUSSION TITLE
Off-Screen rendering on Mac

### DISCUSSION THREAD

**Discussion by Pankaj003** [google/filament#5244] [Q&A]
Hi,
Do latest filament releases supports offscreen rendering on **Mac**?
Currently I am using **v.1.9.9** and offscreen rendering is not available for **Mac** in that.

Thanks,
Pankaj

**Comment by romainguy** [comment]
What kind of offscreen rendering? Filament can render to a texture just fine for instance.

**Reply by Pankaj003** [reply]
Hi, instead of rendering to application window, we need to save the rendering to the file. On Windows we are able to save it to different file sizes but on Mac that do not work and we are forced to save it as per Windows viewport dimension. Is this improved in latest filament releases.

Thanks
